en/docs/deploy/user-management-related-tables.md

```diff
@@ -62,7 +61,7 @@ Following are the columns of the table.
 
 ## UM_MODULE
 
-This table is not being used in Identity Server latest version.
+This table is not being used in the latest version of WSO2 Identity Server.
```

Comment:
can't we remove this info from the docs?

Reply:
IMO, we should let this be till the latest document is tested and restructured.